config/checkstyle/checkstyle.xml

```diff
@@ -37,7 +37,6 @@
 
   <!-- All Java AST specific tests live under TreeWalker module. -->
   <module name="TreeWalker">
-
```

Comment:
Missing line. This may sound nitpicky, but the idea is that unnecessary changes should not be included in a PR.

So the only 3 changes should be green blocks corresponding to the new checks.

Reply:
Other than this, everything else looks good